Automattic/wp-calypso
# Hosting Config: Enable current Cache Clear logic for all users

Related to 1476-gh-Automattic/dotcom-forge

## Proposed Changes

In this PR, I propose to enable all users to clear the cache for their Atomic sites and to refresh the whole block.

<img width="746" alt="image" src="https://user-images.githubusercontent.com/36432/219174290-2d505643-5edf-4f0d-a879-e36e4ea6e92e.png">


## Testing Instructions



1. Navigate to Atomic site, Settings -> Hosting Configuration 
2. Locate the Cache block at the bottom of the page
3. Clear cache
4. Confirm cache was cleared, that button is disabled, and informative text appears
5. Wait a minute and confirm that button is enabled again

## Pre-merge Checklist
